### DISCUSSION TITLE
After Kibana upgrade getting incompatible issues

### DISCUSSION THREAD

**Discussion by jmarkad** [fbaligand/kibana-enhanced-table#328] [General]
Hi when I try to install the plugin it gives the error:
**Plugin enhancedTable [8.9.0] is incompatible with Kibana [7.16.2]**

I thought version 8.9.0 fixed compatibility with Kibana 7.16.0, 7.16.1, 7.16.2 and 7.17.4?

**Comment by fbaligand** [comment]
Well, you have to download the very precise package for Kibana 7.16.2, so this one:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.13.3/enhanced-table-1.13.3_7.16.2.zip
